src/Data/Textile/Step.elm

```diff
@@ -50,6 +50,7 @@ type alias Step =
     , makingWaste : Maybe Unit.Ratio
     , picking : Maybe Unit.PickPerMeter
     , surfaceMass : Maybe Unit.SurfaceMass
+    , dyeingMedium : Maybe Inputs.DyeingMedium
```

Comment:
There's a default medium of "fabric", so is there a reason why we would like to store a `Maybe` here?

Reply:
Adressed in https://github.com/MTES-MCT/ecobalyse/pull/174/commits/c96549a02fb25477323960116be953c8ad0babb4